### DISCUSSION TITLE
Kang Shao produced the first batch

### DISCUSSION THREAD

**Discussion by WYAYZM** [WYAYZM/Hobby-research-community#21] [Polls]
以下批处理出自爱好研究社区康少

<hr /><em>This discussion was created from the release <a href='https://github.com/WYAYZM/Hobby-research-community/releases/tag/V1.0'>Kang Shao produced the first batch</a>.</em>

**Comment by yuchenzhiyi** [comment]
这是什么？

**Reply by WYAYZM** [reply]
批处理
